src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -365,6 +365,25 @@ public function theProductHasVariantPricedAt(
     }
 
     /**
+     * @Given /^(this product) has "([^"]+)" variant priced at ("[^"]+") which does not require shipping$/
+     */
+    public function theProductHasVariantWhichDoesNotRequireShipping(
+        ProductInterface $product,
+        $productVariantName,
+        $price
+    ) {
+        $this->createProductVariant(
```

Comment:
There is no example factory for product variant strictly :/ Do you think it would be worth to implement this? But I think it's a case for separate PR.